clustering/infinispan/extension/src/main/resources/subsystem-templates/infinispan.xml

```diff
@@ -70,7 +70,6 @@
                     <expiration max-idle="100000"/>
                 </local-cache>
                 <invalidation-cache name="entity">
-                    <transaction mode="NON_XA"/>
```

Comment:
This isn't sufficient to affect the default configuration.  You need to modify the following files:
* galleon-pack/src/main/resources/feature_groups/infinispan-dist.xml
* galleon-pack/src/main/resources/feature_groups/infinispan-local.xml

Reply:
@scottmarlow ^^

Reply:
Thanks @pferraro!  Will make that change also.  Thanks @kabir for the ping :)

Reply:
@pferraro please review my update to galleon-pack/src/main/resources/feature_groups/infinispan-dist.xml + galleon-pack/src/main/resources/feature_groups/infinispan-local.xml, to make sure that I didn't remove too much.

Reply:
Looks good now.